test/tap/index.js

```diff
@@ -0,0 +1,64 @@
+const test = require('tap').test
```

Comment:
You don't need to put this file under `test/tap/...`. Just `test/` is fine -- the `test/tap/` thing is an artifact of legacy stuff with npm's tests.